app/src/main/java/org/p2p/wallet/history/ui/history/HistoryPresenter.kt

```diff
@@ -88,11 +88,11 @@ class HistoryPresenter(
             }
         } catch (e: CancellationException) {
             Timber.w(e, "Cancelled history next page load")
+        } catch (e: EmptyDataException) {
```

Comment:
empty data exception ? :) zest'

Reply:
Response model complexity.
But we can workaround.